BootloaderCorePkg/Tools/SblFwUpdate.py

```diff
@@ -0,0 +1,32 @@
+## @ SblFwUpdate.py
+#
+# Copyright (c) 2017-2020, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+import os
+import re
+import sys
+import shutil
+
+def Usage():
+    print "SblFwUpdate"
+    print "Usage:"
+    print "    SblFwUpdate  <Input Capsule File>"
+
+def Main():
+    #
+    # Parse the options and args
+    #
+    argc = len(sys.argv)
+    if argc == 2:
+        shutil.copy(sys.argv[1], '/boot/efi/FwuImage.bin')

```

Comment:
Better copy it to "/boot/FwuImage.bin" since SBL firmware update is not related with EFI?

Reply:
/boot/efi path is required because this way, no changes are needed to SBL code

Reply:
do you mean we hard-coded the path "/boot/efi" in SBL or "/boot/efi" is in the default search path when we only provide capsule name "FwuImage.bin"?

Reply:
In Yocto WIC images the EFI partition is being mapped to '/boot/efi' directory. The default FWU capsule location is SW partition 0 on a FAT32 FS. The script is just copying the FWU capsule file to the EFI partition which happens to align with the default capsule location for all/most of the platforms. The capsule media is set to USB so user would just have to change that to whatever media they are booting Yocto from and this script will work by default.

Reply:
From @jgutbub , it looks this script is specific for the yocto OS that has mapped "/boot/efi" to EFI partition. If other OS doesn't have this mapping, this script might not work. But it is ok for now to make it works for Yocto at least.

Reply:
As mentioned in the commit description, this works only with Linux.

Reply:
In the commit msg, "Copies the capsule to a known location where SBL will look for (/boot/efi/FwuImage.bin)." and here it copied to /boot/efi/FwuImage.bin.
From the SBL code, the SBL didn't really look for "/boot/efi/FwuImage.bin", the capsule file from the config data is only "FwuImage.bin". that's why it is very confusing. 
Anyway, I knows the bkm from James on this since there is a mapping in Yocto.